KitchenLib/Changelogs/v0.7.6.MD

```diff
@@ -0,0 +1,3 @@
+﻿# Release Notes
+
+Changelogs not current at this time, will update later.
```

Comment:
Documentation is important. I think we need to add information to the change log about what exactly this PR is doing.

Reply:
Didn’t have time to write documentation as it was 6am, although with the nature of the platform you can see exactly what was changed in the commit.